src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,16 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} represents an integer.
+     * e.g. -1, 0, 1, 2, 3, ...
+     * Returns true for +1, +2, +3, for consistency
+     * Returns false for any other non-null string input
+     * @param s
+     * @return
+     */
+    public static boolean isNumber(String s) {
+        return s.matches("\\d+(\\.\\d+)?");
+    }
```

Comment:
Very nice regular expression. It would be better if that is kept as a constant named `NUMBER_VALIDATION_REGEX`

Reply:
Ok. I've made and pushed the changes.